lib/ruby_lsp/server.rb

```diff
@@ -290,6 +290,18 @@ def run_initialize(message)
       begin_progress("indexing-progress", "Ruby LSP: indexing files")
 
       global_state_notifications.each { |notification| send_message(notification) }
+
+      if @setup_error
+        message = <<~MESSAGE
+          An error occurred while setting up Bundler. This may be due to a failure when installing dependencies.
+          The Ruby LSP will continue to run, but features related to the missing dependencies will not be available.
```

Comment:
```suggestion
          The Ruby LSP will continue to run, but features related to the missing dependencies will be limited.
```
(otherwise it may sound like the feature gets completely disabled in all places).